lib/DOEKBase/DataAPI/annotation/genome_annotation/ClientAPI.pm

```diff
@@ -96,28 +96,79 @@ sub get_feature_ids {
 
     my %args=@_;
 
-#    print Dumper(@args);
-
-#filters={'type_list':['CDS','mRNA'],'region_list':[],'function_list':[],'alias_list':[]}
-
-#      $self->{'client'}->get_feature_ids($self->{'token'},$self->{'ref'}, {type_list=>['CDS'],'region_list'=>[],'function_list'=>[],'alias_list'=>[]},{});
-
     my $converted_filters = DOEKBase::DataAPI::annotation::genome_annotation::Feature_id_filters->new();
     foreach my $filter (keys %{$args{'filters'}})
     {
         $converted_filters->{$filter}=$args{'filters'}{$filter};
     }
+    if ($args{'filters'}{'region_list'})
```

Comment:
I think that the get_feature_ids() code is still compatible for when PR 133 is rejected.